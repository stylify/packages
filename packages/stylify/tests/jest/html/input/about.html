<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
</head>
<body class="font-family:arial font-size:14px">
	<div class="
		text-align:center
		font-size:48px
		background:black
		max-width:320px
		margin:0__auto
		color:darkred"
	>
		About page
	</div>
</body>
</html>
