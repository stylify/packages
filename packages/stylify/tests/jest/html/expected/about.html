<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
</head>
<body class="s0 s14">
	<div class="
		s2
		s3
		s15
		s5
		s6
		s16"
	>
		About page
	</div>
</body>
</html>
